tests/acceptance/00_basics/03_bodies/inherit_from.cf

```diff
@@ -0,0 +1,54 @@
+##############################################################################
+#
+# Redmine #4309: body inheritance with inherit_from
+#
+##############################################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle agent init
+{
+}
+
+bundle common test
+{
+  reports:
+      "test_inherit" classes => classes_generic_inherit("test_inherit");
+      "test_nonesuch" classes => classes_generic_nonesuch("test_nonesuch");
+      "test_overwrite" classes => classes_generic_overwrite("test_overwrite");
+}
+
+body classes classes_generic_inherit(x)
+{
+      inherit_from => classes_generic;
```

Comment:
This is wrong behavior, IMHO. Here I would expect an error message "Body takes 1 argument but 0 given" or something of the sort. Letting named arguments "bleed through" to the next body without being explicitly specified breaks locality of variables and is unexpected when compared to other languages.